Editor/SceneSetup/XRSimulatorImporter.cs

```diff
@@ -0,0 +1,64 @@
+﻿#if XRIT_1_0_OR_NEWER
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+namespace Innoactive.CreatorEditor.PackageManager.XRInteraction
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    internal class XRSimulatorImporter
+    {
+        public string SimulatorRigPath = null;
+        public string SimulatorPathKey = null;
```

Comment:
What is the PathKey?

Reply:
It is a key used to save and retrieve the path to the XR Rig Simulator prefab in `EditorPrefs`.